wrappers/start-polkadot.sh

```diff
@@ -9,18 +9,20 @@ export SNAP_CURRENT="$(realpath "${SNAP}/..")/current"
 # Create a config if its missing.
 if [ ! -f "${SNAP_COMMON}/service-arguments" ]; then
     echo "==> Creating basic service argument file ${SNAP_COMMON}/service-arguments"
-    echo "--name=Dwellir" > "${SNAP_COMMON}/service-arguments"
+    echo "--name=Unspecified_Dwellir_node" > "${SNAP_COMMON}/service-arguments"
```

Comment:
What would be awesome, is if we could figure out a way to have the instance-name (hostname) go here.